viite-backend/viite-main/src/main/scala/fi/liikennevirasto/viite/dao/LinearLocationDAO.scala

```diff
@@ -289,6 +289,30 @@ class LinearLocationDAO extends BaseDAO {
     fetchByIdMassQuery(ids, rejectInvalids)
   }
 
+
+  /**
+   * Returns the linear locations within the current network (valid_to is null), who
+   * have the given <i>linkId</i>, and
+   * have their M values fit within <i>filterMvalueMin</i>...<i>filterMvalueMax</i> range.
+   *
+   * @param linkId Filters the returned linear locations to those that have this link id.
+   * @param filterMvalueMin Filters the returned linear locations to those having startMvalue at least this much
+   * @param filterMvalueMax Filters the returned linear locations to those having  endMvalue  at most  this much
+   */
+  def fetchByLinkIdAndMValueRange(linkId: String, filterMvalueMin: Double, filterMvalueMax: Double): List[LinearLocation] = {
+    time(logger, "Fetch linear locations by link id, and M values") {
+
+      val query =
+        s"""
+          $selectFromLinearLocation
+          WHERE loc.link_id like '$linkId'
```

Comment:
miksi operaattori "LIKE" eikä "="?

Reply:
link_id on string, en mä oo osannu saada =-merkkiä toimimaan stringien vertailussa  🤔 
Sekö toimis siis? Koulutusta kiitos! 😄

Reply:
Noni, koulutus saatu :D
En kyl tajua, miksen saanut =-merkkiä toimimaan muka, aloitin kyllä sillä...

Mut tossa ei oo tarvetta osastringin vertailulle, niin ehdottomasti "=" kun, kerran toimii; like viittais osastringien vertailuun päin.